datasets/pubmed/README.md

```diff
@@ -0,0 +1,197 @@
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** : [https://www.nlm.nih.gov/databases/download/pubmed_medline.html]()
+- **Documentation:** : [https://www.nlm.nih.gov/databases/download/pubmed_medline_documentation.html]()
+- **Repository:**
+- **Paper:**
+- **Leaderboard:**
+- **Point of Contact:**
+
+### Dataset Summary
+
+NLM produces a baseline set of MEDLINE/PubMed citation records in XML format for download on an annual basis. The annual baseline is released in December of each year. Each day, NLM produces update files that include new, revised and deleted citations. See our documentation page for more information.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+- English
+
+## Dataset Structure
+
+Bear in mind the data comes from XML that have various tags that are hard to reflect
+in a concise JSON format. Tags and list are kind of non "natural" to XML documents
+leading this library to make some choices regarding data. "Journal" info was dropped
+altogether as it would have led to many fields being empty all the time.
+
+The hierarchy is also a bit unnatural but the choice was made to keep as close as
+possible to the original data for future releases that may change schema from NLM's side.
+
+Author has been kept and contains either "ForeName", "LastName", "Initials", or "CollectiveName".
+(All the fields will be present all the time, but only some will be filled)
+
+### Data Instances
+
+```json
+{
+    "MedlineCitation": {
+        "PMID": 0,
+        "DateCompleted": {"Year": 0, "Month": 0, "Day": 0},
+        "NumberOfReferences": 0,
+        "DateRevised": {"Year": 0, "Month": 0, "Day": 0},
+        "Article": {
+            "Abstract": {"AbstractText": "Some abstract (can be missing)" },
+            "ArticleTitle": "Article title",
+            "AuthorList": {"Author": [
+                {"FirstName": "John", "ForeName": "Doe", "Initials": "JD", "CollectiveName": ""}
+                {"CollectiveName": "The Manhattan Project", "FirstName": "", "ForeName": "", "Initials": ""}
+            ]},
+            "Language": "en",
+            "GrantList": {
+                "Grant": [],
+            },
+            "PublicationTypeList": {"PublicationType": [])},
+        },
+        "MedlineJournalInfo": {"Country": "France"},
+        "ChemicalList": {"Chemical": [
+            "RegistryNumber": "XX",
+            "NameOfSubstance": "Methanol"
+        ]},
+        "CitationSubset": "AIM",
+        "MeshHeadingList": {
+            "MeshHeading": [],
+        },
+    },
+    "PubmedData": {
+        "ArticleIdList": {"ArticleId": "10.1002/bjs.1800650203"},
+        "PublicationStatus": "ppublish",
+        "History": {"PubMedPubDate": [{"Year": 0, "Month": 0, "Day": 0}]},
+    },
+}
+```
```

Comment:
it doesn't look like a valid structure here, can you fix that ?

In particular we can see a closing parenthesis alone at `{"PublicationType": [])` and a dictionary with brackets at `[
            "RegistryNumber": "XX",
            "NameOfSubstance": "Methanol"
        ]`

Reply:
It's supposed to be a list of dictionaries.